src/module_wrap.cc

```diff
@@ -387,13 +393,15 @@ void ModuleWrap::GetNamespace(const FunctionCallbackInfo<Value>& args) {
   Local<Module> module = obj->module_.Get(isolate);
 
   switch (module->GetStatus()) {
-    default:
```

Comment:
Is this just for explicitness because this switch/case is now exhaustive and we can count on not having the default?

Reply:
ah no, i meant to add an UNREACHABLE. good catch.